### PR TITLE
Added the Floating action button to Resume to the last read chapter

### PR DESCRIPTION
**Pull request by abhijeetChawla**
- When a chapter is marked as read/unread the lastPageRead is changed to 1,
- fixed a bug which made the boolean expression appear in the ui
- Migrated a file to mui v5 and also removed some unused Styles

### CODE REVIEW

File: src/screens/Manga.tsx [hunk 1]
```diff
@@ -158,7 +179,7 @@ export default function Manga() {
                     overscan={window.innerHeight * 0.5}
                 />
             </LoadingPlaceholder>
-
+            {resumeButton()}
```
Comment (ntbm) [inline src/screens/Manga.tsx:182]: Create a Component here so it reads like <ResumeButton />

**Review by AriaMoradi — CHANGES_REQUESTED** [review]

File: src/screens/Manga.tsx [hunk 2]
```diff
@@ -158,7 +180,7 @@ export default function Manga() {
                     overscan={window.innerHeight * 0.5}
                 />
             </LoadingPlaceholder>
-
+            <ResumeButton />
```
Comment (ntbm) [inline src/screens/Manga.tsx:183]:
```suggestion
            <ResumeFab />
```

**Review by ntbm — CHANGES_REQUESTED** [review]

**Comment by ntbm** [comment]
@AriaMoradi Are we now going with "exactly" where you left of aka `chapters.sort((a, b) => a.lastReadAt - b.lastReadAt)[0]`

**Comment by AriaMoradi** [comment]
> @AriaMoradi Are we now going with "exactly" where you left of aka `chapters.sort((a, b) => a.lastReadAt - b.lastReadAt)[0]`

Can be in another PR

**Pull request merged**